fineract-provider/build.gradle

```diff
@@ -278,6 +280,86 @@ configurations {
 tasks.withType(JavaCompile) {
     options.compilerArgs += ["-Xlint:unchecked","-Xlint:cast","-Werror"] // TODO FINERACT-959 (gradually) enable -Xlint:all (see "javac -help -X")
     options.deprecation = true
+    options.errorprone {
+        disableWarningsInGeneratedCode = true
+        //Todo Remove necessary checks from this list
+        disable("UnusedVariable",
```

Comment:
I see that you have added these so you can enable them and fix gradually rather than fixing the errors at once? Make sure you look at the issues rules you are disabling so that we dont miss catching some critical problems in the system.

Reply:
Yep these are default warning checks and will be removed from this list in separate PR

Reply:
Lets not forget to enable some of the important rules you have disable here in future PRs.